docker-local/README.md

```diff
@@ -66,7 +69,8 @@ npm install
 - A hosts file with the following entries:
 ```
 127.0.0.1       central-ledger.local central-settlement.local ml-api-adapter.local account-lookup-service.local account-lookup-service-admin.local quoting-service.local moja-simulator.local central-ledger central-settlement ml-api-adapter account-lookup-service account-lookup-service-admin quoting-service simulator host.docker.internal
-127.0.0.1 dfspa-backend dfspb-backend pisp-backend dfspa-scheme-adapter dfspb-scheme-adapter pisp-scheme-adapter transaction-request-service
+127.0.0.1 dfspa-backend dfspb-backend pisp-backend dfspa-sdk-scheme-adapter dfspb-sdk-scheme-adapter pisp-sdk-scheme-adapter transaction-request-service
```

Comment:
maybe we should group these resource by parties: pisp, dfspa, dfspb, switch core services